package.json

```diff
@@ -64,7 +64,7 @@
     "meow": "^3.7.0",
     "mkdirp": "^0.5.1",
     "nan": "^2.13.2",
-    "node-gyp": "^3.8.0",
+    "node-gyp": "^6.1.0",
```

Comment:
This can't be updated till v5 because it breaks support for old Node versions. If you need this as part of this setup, please edit the PR to target the v5 branch where it should already be bumped

Reply:
Is there a planned release date for v5?

Reply:
Un-bumped